### PR TITLE
C1 abdelrahman view and edit wishlist

### PR DESCRIPTION
**Pull request by abdelrahmanMaged**
@BeshoSmile @Tharwat7 @Reem-Sharkawy 

Please review and merge

### CODE REVIEW

**Comment by abdelrahmanMaged** [comment]
This is a kind reminder to review the branch 
@Tharwat7 @Reem-Sharkawy @BeshoSmile

**Comment by Reem-Sharkawy** [comment]
#215

**Comment by Reem-Sharkawy** [comment]
the wish list was there but with no posts , and then i refresh , i find the profile without the part for my wish list :D

**Comment by abdelrahmanMaged** [comment]
l2 if you hovered 3la l box you will find the button 3shan asln l styling
bta3 l profile f 7aga f i am waiting for the new profile design

File: project/tager/tager_www/models.py [hunk 1]
```diff
@@ -298,8 +298,8 @@ class Post(models.Model):
     quality_index = models.DecimalField(max_digits=5, decimal_places=2, null=True)
     description = models.CharField(max_length=500, null=True)
     price = models.IntegerField(null=True)
-    edit_date = models.DateField(null=True)
-    pub_date = models.DateField(null=True)
+    edit_date = models.DateField(default = datetime.datetime.now())
```
Comment (BeshoyAtef) [inline project/tager/tager_www/models.py:301]: Wrong logic !!

Comment (abdelrahmanMaged) [inline project/tager/tager_www/models.py:301]: sorry sorry :( i will remove it from the models :D

**Comment by Reem-Sharkawy** [comment]
abdelrahman please update the branch with the new master 
sorry m3lsh , bs 3shan behabel m3aya nw @abdelrahmanMaged

**Comment by abdelrahmanMaged** [comment]
@Reem-Sharkawy done :D

**Comment by Reem-Sharkawy** [comment]
i guess you have to resize the view in which the wish list appears 3shan it seems en homa mawgooden bs msh byneen , and im sure that there are posts 
image

**Comment by abdelrahmanMaged** [comment]
can you elaborate more please as i really do not get what do you mean :(

**Comment by abdelrahmanMaged** [comment]
reem i found the error and i am working on fixing it :)

I am sorry for wasting your time :D
P.s: I know that the cycle has been changed but can you please complete
reviewing this branch only?? or i have to switch to my new reviewer??

**Comment by Reem-Sharkawy** [comment]
lala ana m3akk , w kman 3yza 2shoof hyeb2a 3amel ezay 3shan 2nbaher 
ebhrny :P

**Comment by abdelrahmanMaged** [comment]
@Reem-Sharkawy check again please :D

**Comment by abdelrahmanMaged** [comment]
@Reem-Sharkawy  the final ta3del :D

**Comment by Reem-Sharkawy** [comment]
w b3deeen b2aa , h3od 23mel pull kza mara !!
check the mail btw

**Comment by abdelrahmanMaged** [comment]
leh pull kza mra???

**Comment by HalaKamal** [comment]
tage me when you finish the diagram to review @AbdelrahmanAyad

**Pull request merged**